components/polylith/diff/report.py

```diff
@@ -46,30 +67,3 @@ def print_diff_summary(tag: str, bases: List[str], components: List[str]) -> Non
 
     if bases:
         console.print(f"[base]Changed bases[/]: [data]{len(bases)}[/]")
-
-
-def _changed_projects(
-    projects_data: List[dict], brick_type: str, bricks: List[str]
-) -> set:
-    res = {
-        p["path"].name: set(p.get(brick_type, [])).intersection(bricks)
-        for p in projects_data
-    }
-
-    return {k for k, v in res.items() if v}
-
-
-def print_short_diff(
-    projects_data: List[dict],
-    projects: List[str],
-    bases: List[str],
-    components: List[str],
-) -> None:
-    a = _changed_projects(projects_data, "components", components)
-    b = _changed_projects(projects_data, "bases", bases)
-    c = set(projects)
-
-    res = {*a, *b, *c}
-
-    console = Console(theme=theme.poly_theme)
-    console.print(",".join(res))
```

Comment:
I have possibly made a mistake here by removing this, need to try this out more in detail.